cli/running/running.go

```diff
@@ -2,11 +2,48 @@ package running
 
 import (
 	"fmt"
+	"os"
 
+	log "github.com/sirupsen/logrus"
 	"github.com/tarantool/cartridge-cli/cli/common"
 	"github.com/tarantool/cartridge-cli/cli/project"
 )
 
+func FillCtx(projectCtx *project.ProjectCtx, args []string) error {
+	var err error
+
+	if err := project.SetLocalRunningPaths(projectCtx); err != nil {
+		return err
+	}
+
+	if projectCtx.AppDir == "" {
+		projectCtx.AppDir, err = os.Getwd()
+		if err != nil {
+			return fmt.Errorf("Failed to get current directory: %s", err)
+		}
+	}
+
+	if projectCtx.Name, err = project.DetectName(projectCtx.AppDir); err != nil {
+		return fmt.Errorf("Failed to detect application name: %s", err)
+	}
+
+	projectCtx.StateboardName = project.GetStateboardName(projectCtx)
+
+	if projectCtx.StateboardOnly {
+		projectCtx.WithStateboard = true
```

Comment:
Can we make `return nil` right after this line?

Reply:
I think it's a good approach to check specified instance ID's anyway and warn if they was specified with `--stateboard-only` flag.